### ISSUE TITLE
`diesel migration run` could use some UI improvement on errors

### ISSUE THREAD

**Issue by sgrif** [diesel-rs/diesel#688]
Two quick wins:

It needs to be a bit more obvious when a migration failed to run (literally we should just take up more lines). Additionally, it would be useful for us to give a path to the actual migration file that changed. Right now you just sort of know which one failed because the last line will before the error will be "`Running 873487534`, but going from that to an actual path that you need to edit is a bit annoying.

**Comment by ericho** [comment]
What would the desired output for a failed migration run? Something like :
```
Running migration 20170308040245
Migration on /path/to/migration/up.sql failed with : unrecognized token: "{"
```
or 

```
Running migration 20170308040245
Executing migration script /path/to/migration/up.sql
Failed with : unrecognized token: "{"
```

I'm just wondering what would be the best way and place to put the message.

**Comment by ericho** [comment]
And I also saw #397 should this wait until the refactor is completed?

**Comment by Eijebong** [comment]
I like the first one, but it might not be as good if the error message is very long.

**Comment by Dylan-DPC** [comment]
Hi. Interested in working on this

**Comment by weiznich** [comment]
Fixed by #1790
